config/livewire.php

```diff
@@ -63,16 +63,16 @@
 	*/
 
 	'temporary_file_upload' => [
-		'disk' => null,        // Example: 'local', 's3'              | Default: 'default'
-		'rules' => null,       // Example: ['file', 'mimes:png,jpg']  | Default: ['required', 'file', 'max:12288'] (12MB)
-		'directory' => null,   // Example: 'tmp'                      | Default: 'livewire-tmp'
-		'middleware' => null,  // Example: 'throttle:5,1'             | Default: 'throttle:60,1'
-		'preview_mimes' => [   // Supported file types for temporary pre-signed file URLs...
+		'disk' => null,                      // Example: 'local', 's3'              | Default: 'default'
+		'rules' => null,                     // Example: ['file', 'mimes:png,jpg']  | Default: ['required', 'file', 'max:12288'] (12MB)
+		'directory' => null,                 // Example: 'tmp'                      | Default: 'livewire-tmp'
+		'middleware' => 'throttle:10000,1',  // Example: 'throttle:5,1'             | Default: 'throttle:60,1'
+		'preview_mimes' => [                 // Supported file types for temporary pre-signed file URLs...
 			'png', 'gif', 'bmp', 'svg', 'wav', 'mp4',
 			'mov', 'avi', 'wmv', 'mp3', 'm4a',
 			'jpg', 'jpeg', 'mpga', 'webp', 'wma',
 		],
-		'max_upload_time' => 5, // Max duration (in minutes) before an upload is invalidated...
+		'max_upload_time' => 60,               // Max duration (in minutes) before an upload is invalidated...
```

Comment:
This too. From the throttle above this is presumably per file?

Reply:
yep.